src/goto-symex/symex_builtin_functions.cpp

```diff
@@ -530,37 +518,33 @@ void goto_symext::symex_macro(
 {
   const irep_idt &identifier=code.op0().get(ID_identifier);
 
-  if(identifier==CPROVER_MACRO_PREFIX "waitfor")
-  {
-    #if 0
-    exprt new_fc("waitfor", fc.type());
+  PRECONDITION(identifier == CPROVER_MACRO_PREFIX "waitfor");
```

Comment:
Are we sure this is not a user level error? What would happen if someone fed in a source file with (say) an incorrect macro?